src/codeu/chat/server/RemoteRelay.java

```diff
@@ -59,63 +59,63 @@ public Component(Uuid id, Time time, String text) {
   private static final Serializer<Relay.Bundle.Component> COMPONENT_SERIALIZER =
       new Serializer<Relay.Bundle.Component>() {
 
-    @Override
-    public Relay.Bundle.Component read(InputStream in) throws IOException {
+        @Override
```

Comment:
Looks like all the changes in this file are whitespace only. If so, please revert.